src/pages/use-cases/bgp.jsx

```diff
@@ -0,0 +1,73 @@
+  imageSrc: TravelBee,
+  imageAlt: 'Travel Bee',
+};
+
+const sectionContent1 = {
+  title: 'Elevate your Networking Infrastructure with Cilium’s BGP',
+  description:
+    "Cilium amplifies the power of BGP - the backbone of the internet - to bring you high-speed, scalable, and secure routing for your cloud environments. Seamlessly integrating with existing infrastructure, Cilium's BGP is perfect for a range of deployments, from hybrid to multi cloud to edge. With advanced traffic engineering features, you'll have granular control over your network traffic, optimizing your network's performance and security.",
+  imageSrc: BGPImage1,
+  imageAlt: 'multi-cluster illustration',
+};
+
+const sectionContent2 = {
+  title: 'Effortless Integration of Existing Network Infrastructure',
+  description:
+    "Cilium's BGP support is designed to be simple and easy to integrate with your existing networking infrastructure. Wherever your application is running, Cilium can ensure efficient BGP routing for all your workloads. Many network infrastructures already use BGP for routing. By leveraging BGP, Cilium can  seamlessly integrate with existing infrastructure, allowing Kubernetes pods to communicate with other parts of the network.",
+  videoSrc: 'https://www.youtube.com/embed/Nzh2jc6qW6Y',
+  whiteBackground: true,
+};
+
+const testimonials = [
+  {
+    logo: <TripLogo />,
+    title: "Trip.com's Hybrid Cloud Approach with Cilium for Workload Routing",
+    CTAtext: 'Read The Blog Post',
+    url: 'https://cilium.io/blog/2020/02/05/how-trip-com-uses-cilium/',
+    description:
+      'Trip.com chose a topology based on where workloads run. Bare-metal on-premises workloads use Direct routing via BGP using BIRD, while AWS workloads use AWS ENI routing mode via Cilium AWS ENI support. This topology provides native routing architecture in both the bare-metal and AWS environments with all its performance benefits. In addition, it provides a universal Kubernetes services and Network Policy model across all environments to minimize the differences between the bare-metal and cloud environments.',
+  },
+  {
+    title: 'How IKEA leverages cilium’s BGP ',
+    logo: '',
+    CTAtext: 'Watch The Talk',
+    url: 'https://clickhouse.com/blog/building-clickhouse-cloud-from-scratch-in-a-year',
+    description:
+      'We run all our Kubernetes nodes as routers that advertises k8s pods IP’s to our data center network. We run our nodes with 4 network interfaces in 2 pairs (ECMP). 1 pair for frontend application traffic, 1 pair for backend replication/data traffic. This provides us with flexibility, scalability, redundancy and global routable pod IPs.',
+    quote: true,
+    person: 'Karsten Nielson - Senoir systems engineer, Ikea Retail',
```

Comment:
```suggestion
    person: 'Karsten Nielson - Senoir Systems Engineer, Ikea Retail',
```

Reply:
There's a bad URL here, I'll correct it.

Reply:
@paularah Todo

Reply:
senior is spelt wrongly too